.travis.yml

```diff
@@ -1,4 +1,8 @@
 language: node_js
+sudo: false
+cache:
+  directories:
+    - node_modules
```

Comment:
what advantage does this give us? won't it be a problem when we try to update a dep?